Doom-Mummies/Welcome-to-Nashville
# Mj itinerary form

# Description

Added itinerary HTML to the DOM in itinerary.js and empty span added to save itinerary results

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. ``git fetch --all``
2. ``git checkout mj-itineraryForm``
3. check that itinerary shows on browser



# Checklist:
